sycl/source/detail/device_info.hpp

```diff
@@ -746,6 +762,8 @@ struct get_device_info_impl<
            matrix_type::fp32, matrix_type::fp32},
           {8, 0, 0, 0, 16, 16, matrix_type::bf16, matrix_type::bf16,
            matrix_type::fp32, matrix_type::fp32},
+          {8, 0, 0, 0, 16, 8, matrix_type::tf32, matrix_type::tf32,
```

Comment:
This doesn't relate to GNR, can you please update the caption/description of PR?